messages/dev.convert.messages.md

```diff
@@ -10,6 +10,15 @@ Converts a .json messages file to markdown. Preserves the filename and the origi
 
 Filename to convert
 
+# flags.project-dir.summary
+
+Location project where messages are to be audited.
+
+# flags.project-dir.description
+
+The project directory.
```

Comment:
Please remove this because it adds no value.

Reply:
@jshackell-sfdc Going to have to wait until cmd ref is fixed.